manifests/04-rbac-rolebinding.yaml

```diff
@@ -37,3 +37,30 @@ subjects:
 - kind: ServiceAccount
   name: console-operator
   namespace: openshift-console-operator
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: openshift-config-reader
```

Comment:
`name: console-operator` 
` oc get rolebinding -n openshift-console` should probably list `console-operator` as a rolebinding.